README.md

```diff
@@ -1,6 +1,7 @@
 # Python Docs Sphinx Theme
 
 This is the theme for the Python documentation.
+It requires Python 3.9 or newer and Sphinx 3.4 or newer.
```

Comment:
In another project we noticed the Python version mentioned in docs said 3.6+, but in reality it's 3.9+ and we'd forgotten to update that three times!

To DRY, perhaps we don't need to document this in words, seeing as we have definitive metadata in `pyproject.toml`? 

That is:

```toml
requires-python = ">=3.9"
...
dependencies = [
  "sphinx>=3.4",
]
```

Reply:
I suppose I put it in the README to make it very clear -- perhaps we could replace it with a policy statement instead? I.e.:

"It supports all Sphinx versions required by the [stable](https://devguide.python.org/versions/) Python branches."

I'm not a huge fan of this wording, though -- it feels clunky.

Reply:
Well, it's okay to leave this in the README if you like, at least it's the repo's first page and not some other "hidden"/forgotten docs page.